a3p-integration/proposals/g:3-ymax-alpha2/package.json

```
@@ -0,0 +1,34 @@
+{
+  "agoricProposal": {
+    "source": "subdir",
+    "$comment": "board013515 originates from mainnet proposal 100",
+    "sdk-generate": [
+      "../../portfolio-deploy/src/portfolio.build.js submission --replace board013515"
+    ],
+    "type": "/agoric.swingset.CoreEvalProposal"
+  },
+  "type": "module",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/client-utils": "dev",
+    "@agoric/fast-usdc": "dev",
+    "@agoric/synthetic-chain": "0.5.5",
```

```suggestion
    "@agoric/synthetic-chain": "0.5.8",
```
or whatever is latest :)